src/components/Fields/TextArea.js

```diff
@@ -98,6 +98,29 @@ class TextArea extends Component {
         }
     }
 
+    componentDidUpdate(prevProps) {
+        const { hasFocus } = this.state;
+        const { tooltipError, isValid } = this.props;
+
+        const isShowingErrorTooltip = !isValid && !!tooltipError;
+        const wasShowingErrorTooltip = !prevProps.isValid && !!prevProps.tooltipError;
+        if (
+            (hasFocus && isShowingErrorTooltip && !wasShowingErrorTooltip) ||
+            (hasFocus && !isShowingErrorTooltip && wasShowingErrorTooltip)
+        ) {
+            this.fixInputFocus();
+        }
+        autosize.destroy(this.inputRef);
+        autosize(this.inputRef);
```

Comment:
We don't want to reset `autosize` on every component update. It's only needed when the tooltip is shown or hidden - so within the if block above. And I'd probably call `fixInputFocus` last to ensure the focus is not affected by the `autosize` reset.

Reply:
@jfrej  Changed as suggested